common-libs/common/src/services/DidDocumentService/ElemDidDocument.ts

```diff
@@ -78,36 +77,15 @@ export default class ElemDidDocument {
     }
   }
 
-  private _buildDIDDocModel(
-    seedHex: string,
-    publicKeyPrimary: string = null,
-    privateKeyPrimary: string = null,
-    publicKeyRecovery: string = null,
-    privateKeyRecovery: string = null,
```

Comment:
So apparently the old version of this method didn't actually use private keys. @DenisPopov15 , could it be that there was some bug and private keys should be somehow used here after all?

Reply:
@inga-affinidi yeap, Probably its just over required param here (so not really used), the only place where potential its could be required here - its to sign didDocument